### DISCUSSION TITLE
FPS Increments

### DISCUSSION THREAD

**Discussion by ANIMISM-LTD** [rewbs/sd-parseq#92] [General]
Would be good to see FPS value reflected on the graph rather than it being stuck on 20fps increments 

So if say we set at 30fps the graph lines would be spaced at 30, 60, 90...

If decimal places added could have values 29.97, 59.94, 89.91...

Would be great to see this feature, will definitely buy you a coffee! ☕️

**Comment by rewbs** [comment]
Hi, that's a good idea, I have raised it as a feature request here: https://github.com/rewbs/sd-parseq/issues/97
